packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -1070,8 +1071,8 @@ export default function ImportTransactions({ modalProps, options }) {
 
 function getParseOptions(fileType, csvOptions, ofxOptions) {
   if (fileType === 'csv') {
-    const { csvDelimiter, hasHeaderRow } = csvOptions;
-    return { csvDelimiter, hasHeaderRow };
+    const { delimiter, hasHeaderRow } = csvOptions;
+    return { delimiter, hasHeaderRow };
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏ah.. sorry for noticing this only now. But we are currently still using `csvDelimiter` in some places and `delimiter` in others. So by applying this change we're breaking the app in other places. Would you mind updating all the places to use either one or the other (search in this same file)?